api/views/slack_helpers.py

```diff
@@ -229,18 +264,33 @@ def get_message(data: Dict) -> str:
     """
     event = data.get("event")
     # Note: black and flake8 disagree on the formatting. Black wins.
-    return event["text"][event["text"].index(">") + 2 :].lower()  # noqa: E203
+    try:
+        return event["text"][event["text"].index(">") + 2 :].lower()  # noqa: E203
```

Comment:
The docstring explains what this does and why it's set up -- strings come in that look like `"<@UTPFNCQS2> hello!"` because we have to preface all commands with "@blossom", so this just strips out that first identifier and returns the rest so that we can work with it.